examples/sandbox/src/App.js

```diff
@@ -173,7 +173,7 @@ export default function App() {
               }
               bodyHeader={
                 <React.Fragment>
-                  <PagingInfo />
+                  {wasSearched && <PagingInfo />}
                   <ResultsPerPage />
```

Comment:
@byronhulcher One small thing since you're in here making updates anyway. Could you update this line to...

```
{wasSearched && <ResultsPerPage />}
```

That should give us a nice clean loading experience. Right now the ResultsPerPage pops in when you start typing text in the sandbox.

![latest](https://user-images.githubusercontent.com/1427475/56499729-a3c63b80-64d5-11e9-8d2b-61492118496c.gif)